### DISCUSSION TITLE
updating results for biokg

### DISCUSSION THREAD

**Discussion by afshinsadeghi** [snap-stanford/ogb#255] [General]
Hi, at the presentation of GFA-NN the nice ECML audience suggested reporting the new result for biokg to you. Here they are on page 13. :) 

https://2021.ecmlpkdd.org/wp-content/uploads/2021/07/sub_1096.pdf

the code: https://github.com/afshinsadeghi/GFA-NN

Cheers,
Afshin

**Comment by weihua916** [comment]
Hi! Thanks for your leaderboard contribution. Please submit your results via [this page](https://ogb.stanford.edu/docs/leader_overview/), and we will update the leaderbaord.
